client/src/lib/session/components/Slides/Slides/Blocks/Lottie.tsx

```diff
@@ -91,6 +92,11 @@ const Lottie: React.FC<LottieProps> = ({
     [audioSource],
   );
 
+  useEffect(() => {
+    IdleTimerManager.setIdleTimerDisabled(active);
+    return () => IdleTimerManager.setIdleTimerDisabled(false);
+  }, [active]);
```

Comment:
Could we make it enabled only when it's actually playing?
![image](https://media.tenor.com/L4c21F3eAR0AAAAC/dead-battery.gif)